Playground.js

```diff
@@ -39,15 +39,15 @@ Playground.prototype.contains = function (id) {
   return this._codeObjects.has(id);
 };
 
-Playground.prototype.getData = function (property) {
+Playground.prototype.getData = function () {
   return Array.from(this._codeObjects.values())
-    .map((codeObject) => codeObject.getData(property));
+    .map((codeObject) => codeObject.getData());
 };
 
-Playground.prototype.getDataFor = function (codeObjectId) {
-  var codeObject =
-    this._codeObjects.get(codeObjectId) ||
-    new CodeObject(codeObjectId);
+Playground.prototype.getDataByName = function (codeObjectName) {
```

Comment:
If we’re looking up objects by name now… we might as well call it an id 🙄

Reply:
To me, this is following the technical id pattern. It's normal to have a "getByFunctionalKey()" function.